blinky.core

```diff
@@ -77,6 +82,15 @@ targets:
         pnr: next
     toplevel : blinky
 
+  ulx3s:
+    default_tool : trellis
+    filesets : [rtl, proginfo, ulx3s]
+    parameters : [clk_freq_hz=25000000]
+    tools:
+      trellis:
+        nextpnr_options : [--45k --basecfg ../src/fusesoc_utils_blinky_0/ulx3s/ulx3s_empty.config --lpf ../src/fusesoc_utils_blinky_0/ulx3s/ulx3s_v20.lpf]
```

Comment:
Instead of pointing to `./src/fusesoc_utils_blinky_0/ulx3s/ulx3s_empty.config`, we can add the option `copyto` for the config file `ulx3s/ulx3s_empty.config : {file_type : user, copyto : ulx3s_empty.config}` That way the path won't break if the path to the source tree change.

Let me know if you want to send an updated patch, or if I should just amend this locally. Happy with either

Reply:
I realized I didn't really explain how copyto works. It will copy the file to a new file relative to the work root with the specified name, so that we can just reference it with `--basecfg ulx3s_empty.config`

Reply:
Looking again, the same would apply to the lpf file